client/components/backup-retention-management/hooks.tsx

```diff
@@ -28,50 +14,9 @@ export function useEstimatedCurrentSiteSize(): number {
  * @param desiredRetentionDays   Desired backup retention days
  * @returns Space needed in bytes
  */
-export function useEstimateSpaceNeeded(
+export function getSpaceNeededInBytes(
```

Comment:
Do we really need a function that multiplies 2 numbers? I think we can just replace it with a simple multiplication where necessary, can't we?

Reply:
We were anticipating to future changes, but as you said previously, it is better to follow YAGNI principle. If we needed in the future, then we will create those functions again.

Reply:
We just removed both functions and its tests :)